tests/integration/seed/seedsVerification.test.js

```diff
@@ -174,6 +195,184 @@ describe('SEEDS VERIFICATION', () => {
               ));
           expect(isEveryTraineeCompanyInAccessRules).toBeTruthy();
         });
+
+        it('should pass if every subprogram exists', () => {
+          const coursesExist = courseList.map(course => course.subProgram).every(subProgram => !!subProgram);
+          expect(coursesExist).toBeTruthy();
+        });
+
+        it('should pass if every blended course has companies field', () => {
+          const everyBlendedCourseHasCompanies = courseList
+            .filter(course => course.format === BLENDED)
+            .every(course => has(course, 'companies'));
+          expect(everyBlendedCourseHasCompanies).toBeTruthy();
+        });
+
+        it('should pass if every company exists', async () => {
+          const someCompaniesDontExist = courseList
+            .filter(c => c.format === BLENDED)
+            .some(c => c.companies.some(company => company === 'company not found'));
+
+          expect(someCompaniesDontExist).toBeFalsy();
+        });
+
+        it('should pass if none course has company in duplicate', () => {
+          const someCompaniesAreInDuplicate = courseList
+            .filter(course => get(course, 'companies.length'))
+            .some((course) => {
+              const companiesWithoutDuplicates = [...new Set(course.companies.map(c => c._id.toHexString()))];
+
+              return course.companies.length !== companiesWithoutDuplicates.length;
+            });
+
+          expect(someCompaniesAreInDuplicate).toBeFalsy();
+        });
+
+        it('should pass if every intra course has company', () => {
+          const everyIntraCourseHasCompany = courseList
```

Comment:
if intra course has one and only one company ?

Reply:
done